src/chains.json

```diff
@@ -0,0 +1,4124 @@
+    "2713017997578000": "dchaint",
+    "2716446429837000": "dchainmainnet",
+    "0x534e5f4d41494e": "sn",
+    "0x534e5f5345504f4c4941": "sn-sep"
+  },
+  "SHORTNAME_TO_CHAIN_ID": {
+    "eth": "1",
+    "exp": "2",
+    "rop": "3",
+    "rin": "4",
+    "gor": "5",
+    "kot": "6",
+    "tch": "7",
+    "ubq": "8",
+    "tubq": "9",
+    "oeth": "10",
+    "meta": "11",
+    "kal": "12",
+    "dstg": "13",
+    "flr": "14",
+    "diode": "15",
+    "cflr": "16",
+    "tfi": "17",
+    "TST": "18",
+    "sgb": "19",
+    "esc": "20",
+    "esct": "21",
+    "eladid": "22",
+    "eladidt": "23",
+    "kardiachain": "24",
+    "cro": "25",
+    "L1test": "26",
+    "shib": "27",
+    "BobaRinkeby": "28",
+    "L1": "29",
+    "rsk": "30",
+    "trsk": "31",
+    "GooDT": "32",
+    "GooD": "33",
+    "scai": "34",
+    "tbwg": "35",
+    "dx": "36",
+    "xpla": "37",
+    "val": "38",
+    "u2u": "39",
+    "TelosEVM": "40",
+    "TelosEVMTestnet": "41",
+    "lukso": "42",
+    "pangolin": "43",
+    "crab": "44",
+    "pangoro": "45",
+    "darwinia": "46",
+    "aic": "47",
+    "etmp": "48",
+    "etmpTest": "49",
+    "xdc": "50",
+    "txdc": "51",
+    "cet": "52",
+    "tcet": "53",
+    "OP": "54",
+    "ZYX": "55",
+    "bnb": "56",
+    "sys": "57",
+    "OntologyMainnet": "58",
+    "eos-legacy": "59",
+    "go": "60",
+    "etc": "61",
+    "tetc": "62",
+    "metc": "63",
+    "ellaism": "64",
+    "tokt": "65",
+    "okt": "66",
+    "dbm": "67",
+    "SO1": "68",
+    "okov": "69",
+    "hsc": "70",
+    "cfxtest": "71",
+    "dxc": "72",
+    "FNCY": "73",
+    "idchain": "74",
+    "DSC": "75",
+    "mix": "76",
+    "spoa": "77",
+    "primuschain": "78",
+    "zenith": "79",
+    "GeneChain": "80",
+    "joc": "81",
+    "Meter": "82",
+    "MeterTest": "83",
+    "linqto-devnet": "84",
+    "gttest": "85",
+    "gt": "86",
+    "nnw": "87",
+    "vic": "88",
+    "vict": "89",
+    "gar-s0": "90",
+    "gar-s1": "91",
+    "gar-s2": "92",
+    "gar-s3": "93",
+    "sdlt": "94",
+    "camdl": "95",
+    "bkc": "96",
+    "bnbt": "97",
+    "six": "98",
+    "poa": "99",
+    "gno": "100",
+    "eti": "101",
+    "tw3g": "102",
+    "WLC": "103",
+    "tklc": "104",
+    "dw3g": "105",
+    "vlx": "106",
+    "ntn": "107",
+    "TT": "108",
+    "shibariumecosystem": "109",
+    "xpr": "110",
+    "ETL": "111",
+    "coinbit": "112",
+    "deh": "113",
+    "c2flr": "114",
+    "debank-testnet": "115",
+    "debank-mainnet": "116",
+    "auptick": "117",
+    "arcology": "118",
+    "enuls": "119",
+    "enulst": "120",
+    "REAL": "121",
+    "fuse": "122",
+    "spark": "123",
+    "dwu": "124",
+    "OYchainTestnet": "125",
+    "OYchainMainnet": "126",
+    "feth": "127",
+    "heco": "128",
+    "Innovator": "129",
+    "unichain": "130",
+    "tgram": "131",
+    "nfic": "132",
+    "HSKT": "133",
+    "rlc": "134",
+    "AlyxTestnet": "135",
+    "deam": "136",
+    "pol": "137",
```

Comment:
This may require some change in our UI, to use `pol` instead of `matic`

Reply:
Added compatibility for matic for now (Safe still uses `matic` for example).

If you use `pol` and `matic` it will both map to 137.
If you use 137 it will map to matic.